app/packages/looker/src/overlays/util.ts

```diff
@@ -88,7 +97,7 @@ export function isRgbMaskTargets(
 }
 
 // Return true is string is a valid color
-export function isValidColor(color: string): boolean {
+export function isValidColor(color: string | undefined | null): boolean {
```

Comment:
why do you need both `undefined` and `null`

Reply:
I was trying to remove the eslint type warnings. Sometimes we pass in `null` to this function as a condition check